src/server/components/lists/controllers/index.ts

```diff
@@ -92,11 +121,40 @@ export function listsGetController(req: Request, res: Response): void {
   }
 
   const { serviceType } = params;
+  let { languagesProvided, servicesProvided } = params;
 
   let questionsSequence: QuestionName[];
   let partialPageTitle: string = "";
+  let partialPageHintText: string = "";
   let partialToRender: string = "";
   let error: boolean | QuestionError = false;
+  let partialData: QuestionDataSet[] | QuestionData[];
+  let languagesRows, languageNamesProvided, serviceNamesProvided;
+  let backUrl: string = "";
+
+  if (languagesProvided) {
+    const cleanedLanguagesProvided = cleanLanguagesProvided(languagesProvided as string);
+    languagesProvided = cleanedLanguagesProvided;
+    params.languagesProvided = cleanedLanguagesProvided ?? undefined;
+    languagesRows = getLanguagesRows(languagesProvided as string, queryString);
+    const paramsCopy = { ...params };
+    delete paramsCopy.languagesPopulated;
```

Comment:
you can "remove" a key when you're destructuring

```
const { languagesPopulated, ...rest } = params;
console.log(rest) // all of the keys without languagesPopulated
```

you can combine this with some of my comments above for even more concise code 💪 

```
const newParams = {...rest, langaguesProvided: cleanLanguagesProvided(params.langaugesProvided}
```